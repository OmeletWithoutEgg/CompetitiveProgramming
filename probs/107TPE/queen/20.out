557618.78 -46546111.88
