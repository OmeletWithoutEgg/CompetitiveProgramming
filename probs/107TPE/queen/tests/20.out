4157026.18 8743944.91
